### ISSUE TITLE
No upload possible on Android

### ISSUE THREAD

**Issue by derOtto** [root-gg/plik#348]
## Expected Behavior
Uploading files on Android browsers should work by clicking on _Add files_ button.

## Current Behavior
It's not (easy) possible to upload files on a browser under Android. You could only start to take a picture but not access the storage.

## Possible Solution
Removing the two lines [128](https://github.com/root-gg/plik/blob/master/webapp/partials/main.html#L128) and [174](https://github.com/root-gg/plik/blob/master/webapp/partials/main.html#L174) from plik/webapp/partials/main.html has fixed it for me. There is no missing function on Android (you can still start an action to take a picture with cam) but you could also upload any other file from storage too.

_Don't know if this produce any problems on iOS._

Maybe it should be set depending on the operating system / user agent.

**Comment by sebPomme** [comment]
Hi,
Just to add some information.

I have the same problem, I try your fix 
```dockerfile
FROM rootgg/plik:latest

RUN sed -i '/camera/d' /home/plik/webapp/dist/partials/main.html
```
and that won't fix the problem on my main web browser (opera 62.3.3146.57763). 
But fix it on chrome 88.0.4324.181.

**Comment by dtuando** [comment]
OMG thank you! Ive been waiting for the supposed fix in 1.3.5... this fixed it for me.


**What I did.**
I removed lines 128 and 174. 
Refreshed my phone browser (Pixel 5 Chrome) and it worked!

Opened up my iPad to test in safari and firefox, but I Couldnt see the upload box or button.

I got back and add lines 174 and 128 back and bam... issue gone.. and now it works on all my devices.

**Comment by sebPomme** [comment]
I just try with 1.3.5 and it's seem fixed.
